packages/create-playwright/assets/playwright.config.js

```diff
@@ -8,13 +8,16 @@ const path = require('path');
  */
 const config = {
   // Timeout per test
-  timeout: 30 * 1000,
+  //timeout: 30 * 1000,
```

Comment:
- Empty line between comments and commented code, or different style of comments (/* */)
- Space missing after //